diff --git a/dist/index.js b/dist/index.js
index b4fbd644ea9430b83addd30a1adb9c092fdb3941..ad09b4ad5e8d2d72161a9a3242623134b4900071 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -360,7 +360,7 @@ var formatWriteFilePath = (file, options) => {
   }
   return formatWriteFileName(path4);
 };
-var formatWriteFileName = (file) => `${file}${file.endsWith("d.ts") ? "" : ".d.ts"}`;
+var formatWriteFileName = (file) => file.replace(/\.css$/, ".d.css.ts");
 var formatExportTypeFileName = (file) => basename(file.replace(".ts", ""));
 var ensureDirectoryExists = async (file) => {
   await mkdir(dirname(file), { recursive: true });
